core/service.go

```diff
@@ -151,18 +162,59 @@ func (s *Service) StorageRoot() (common.Hash, error) {
 	return s.storageState.StorageRoot()
 }
 
+func (s *Service) handleBabeSession() {
+	for {
+		<-s.epochDone
+		log.Trace("CORE: BABE epoch complete, initializing new session")
```

Comment:
I like the specification of the package here in the logs. I feel like we have discussed this and we should be doing this everywhere. I am not sure about the all caps `CORE` for formatting though and formatting in general for this log looks a bit out of place alongside the other logs.

Reply:
yeah I'm not totally sure how to format logs like this either, in substrate the packages names are capitalized in the logs so I went off that, but I'm fine with changing it